### PR TITLE
[django] Improve support for django 2.2 #231

### LINKED ISSUE #231: [django] Add support for django 2.2
Add support for django 2.2.

### PR DESCRIPTION
**Pull request by Vivekrajput20**
closes #231

### CODE REVIEW

**Comment by Vivekrajput20** [comment]
@nemesisdesign  Please revert the changes made in the previous PR and review this one.

File: django_freeradius/base/admin.py [hunk 1]
```diff
@@ -141,16 +142,32 @@ def has_delete_permission(self, request, obj=None):
             return False
         return super().has_delete_permission(request, obj)
 
-    def delete_selected_batch(self, request, queryset):
+    def delete_selected_groups(self, request, queryset):
```
Comment (nemesisdesign) [inline django_freeradius/base/admin.py]: See here: https://stackoverflow.com/questions/10814972/override-delete-selected-and-update-fields-of-child-model

Maybe we can simply name it `delete_selected`. What do you think?

File: django_freeradius/base/admin.py [hunk 2]
```diff
@@ -141,16 +142,32 @@ def has_delete_permission(self, request, obj=None):
...
+        if n:
+            queryset.delete()
+            self.message_user(request, _("Successfully deleted %(count)d %(items)s.") % {
+                "count": n, "items": model_ngettext(self.opts, n)
+            }, messages.SUCCESS)
+        return None
+
+    delete_selected_groups.allowed_permissions = ('delete',)
+
+    def get_actions(self, request):
```
Comment (nemesisdesign) [inline django_freeradius/base/admin.py]: if my previous suggestion is correct, we may be able to remove this

File: django_freeradius/base/admin.py [hunk 3]
```diff
@@ -273,6 +290,12 @@ def delete_model(self, request, obj):
         obj.users.all().delete()
         super(AbstractRadiusBatchAdmin, self).delete_model(request, obj)
 
+    def get_actions(self, request):
```
Comment (nemesisdesign) [inline django_freeradius/base/admin.py]: same here

File: django_freeradius/base/admin.py (hunk 1 above)
Comment (Vivekrajput20) [reply django_freeradius/base/admin.py]: I had tried to overwrite it but with the introduction of [admin.E130](https://docs.djangoproject.com/en/2.2/ref/checks/#modeladmin) we can't name any other action 'delete selected' and I had no choice but to rename the custom action.

**Comment by Vivekrajput20** [comment]
@nemesisdesign I tried calling Django's default 'delete_selected' inside 'delete_selected_groups' but Idk why it didn't work. So I had to mostly rewrite it in  'delete_selected_groups'.

I would suggest review it and test it asap as the last PR that you merged is buggy. It should be reverted back or changed asap.

Vivek

**Pull request merged**